templates/catalog/section.html

```diff
@@ -0,0 +1,58 @@
+{% extends 'layout/inner_page.html' %}
+
+{% block content %}
+  {% include 'components/breadcrumbs.html' with page=page base_url=base_url only %}
+  {% include 'components/page_h1.html' with page=page only %}
+
+  <section class="category-search">
+    <div class="container container-fluid">
+      <div class="row center-xs">
+        <div class="col-xs-12 col-md-8">
+          <div class="search">
+            <input class="search-field" id="search-filter" type="text"
```

Comment:
Probably we should implement the search for this, let's create a relevant todo

Reply:
i think we should not. I already created the same subtask for Series page with the same functionality. Then i cancelled it.
https://github.com/fidals/stroyprombeton/issues/584
Seems search feature is not required for pages like Series or Section

Reply:
i'll remove search html from section and series